service/index.js

```diff
@@ -25,12 +26,12 @@ const getProfile = (email) => {
     });
 };
 
-const putEditProfile = (email, first_name, last_name, age, snack, animal_type) => {
+const putEditProfile = (user_id, first_name, last_name, age, snack, animal_type) => {
   return pool
     .connect()
     .then((client) => {
-      const query = 'UPDATE users SET first_name = $2, last_name = $3, age = $4, snack = $5, animal_type = $6 WHERE email = $1';
-      const values = [email, first_name, last_name, age, snack];
+      const query = 'UPDATE users SET first_name = $2, last_name = $3, age = $4, snack = $5, animal_type = $6 WHERE user_id = $1';
```

Comment:
I had not seen this "$" syntax before, but I 'll have to give it a try the next time I'm in Postgres!

Reply:
It's really cool, similar to VALUES "?" in mysql